src/mame/drivers/dooyong.cpp

```diff
@@ -755,9 +770,9 @@ uint32_t popbingo_state::screen_update_popbingo(screen_device &screen, bitmap_in
 
 	for (int y = cliprect.top(); cliprect.bottom() >= y; y++)
 	{
-		uint16_t const *const bg_src(&m_bg_bitmap[0].pix16(y, 0));
-		uint16_t const *const bg2_src(&m_bg_bitmap[1].pix16(y, 0));
-		uint16_t *const dst(&bitmap.pix16(y, 0));
+		const u16 *const bg_src(&m_bg_bitmap[0].pix16(y, 0));
+		const u16 *const bg2_src(&m_bg_bitmap[1].pix16(y, 0));
```

Comment:
This is exactly the kind of place where this change in style is problematic.  The first `const` applies to the thing immediately to its right, while the other `const` refers to the thing to its left.  it's easier to misinterpret code like this.